libs/ui/elements/src/lib/kind-badge/kind-badge.component.ts

```diff
@@ -40,6 +40,7 @@ enum KindConfig {
   imports: [NgIconsModule, CommonModule, BadgeComponent, TranslateModule],
```

Comment:
Not coming from this PR, but is not used anymore:
```suggestion
  imports: [NgIconsModule, CommonModule, TranslateModule],
```

Reply:
good point!
how do you find unused dep and components so easily 👀?

Reply:
When I take a look in vs code, it highlights them to me :-)